assets/components/checkoutHeading/checkoutHeading.scss

```diff
@@ -0,0 +1,25 @@
+.component-checkout-heading {
+  color: #fff;
+
+  .component-left-margin-section__content {
+    background-color: gu-colour(garnett-neutral-1);
+    padding: 0 0 4px ($gu-h-spacing / 2);
+  }
+
+}
+
+.component-checkout-heading__heading {
+  @include gu-fontset-heading-large;
+  margin-bottom: $gu-v-spacing * 3;
+  width: 300px;
```

Comment:
I put in a `gu-span()` mixin mirroring dotcom that will calculate these sizes (number of columns) more neatly for us, (this would be 4 and 6 columns in this case)